Documentation/TypoScript/Index.rst

```diff
@@ -0,0 +1,170 @@
+    :ref:`Blocks in the TypoScript syntax <t3coreapi:typoscript-syntax-syntax-curly-brackets>`.
+#.  Another comment.
+#.  Assigns a content object (also called "cObject") of type
+    :ref:`TEXT <t3tsref:cobj-text>` to
+    :confval:`index number 10 <t3tsref:page-array>` of `page`. It has the path
+    `page.10` if you want to change it later.
+#.  Assigns the value `Hello, world.` to the `value` property of the TEXT
+    cObject stored in path `page.10`.
+
+Clear all caches via the following console command or the button in the backend:
+
+..  code-block:: bash
+
+    ddev typo3 cache:flush
+
+You can now preview the result.
+
+..  todo: Add link on how to preview a page.
+..  todo: Link to how to clear all caches
+
+..  _typoscript-hello-world-result:
+
+Resulting web page
+==================
+
+Here is the resulting web page HTML source for both the TypoScript-only and
+the Fluid based implementations. Notice how TYPO3 has added default markup
+around the single line of content:
+
+..  literalinclude:: _minimal-output.html
+    :caption: Example frontend output
+
+Debug the TypoScript in the backend module "Active TypoScript"
+==============================================================
+
+Open the backend module
+:guilabel:`Site Management > TypoScript > Active TypoScript`.
+
+..  figure:: ActiveTypoScript.png
+    :alt: Screenshot of the submodule "Active TypoScript" in the module "Site Management > TypoScript"
+
+    Switch the submodules of module :guilabel:`Site Management > TypoScript`
+    using the Dropdown in the module header.
+
+You can find the variable `page` that you just defined. The other variables have
+been created by TypoScript loaded globally by the TYPO3 Core and system
+extensions. They are some of the
+:ref:`Reserved top-level objects <t3tsref:top-level-objects-other-reserved-tlo-s>`.
+
+..  _typoscript-play-around:
+
+Play around with TypoScript
+===========================
+
+You can now try out a couple of TypoScript commands to familiarize yourself
+with TypoScript.
+
+Here are some examples:
+
+..  _typoscript-example-stdwrap:
+
+Wrap "Hello, world." in p-tags
+------------------------------
+
+..  literalinclude:: _stdwrap.typoscript
+    :caption: config/sites/main/setup.typoscript
+    :linenos:
+    :emphasize-lines: 4-7
+
+4.  As we now have several options for :ref:`TEXT <t3tsref:cobj-text>` object
+    with path `page.10`, we switch to the
+    :ref:`block syntax <t3coreapi:typoscript-syntax-syntax-curly-brackets>` here.
+5.  Assign the text to the :confval:`value <t3tsref:text-value>` property of
+    the TEXT object.
+6.  We use the :confval:`stdWrap <t3tsref:text-stdwrap>` property of the TEXT
+    object to configure the :ref:`stdWrap <t3tsref:stdwrap>` function.
+
+    In this function we use option wrap :confval:`wrap <t3tsref:stdwrap-wrap>`.
+    It surrounds the current content of the TEXT object as set in line 5 with
+    the value defined here. The pipe `|` character is replaced with the text.
```

Comment:
```suggestion
    the value defined here. The pipe `|` character is replaced with the text stored against the value property.
```

Reply:
"Stored against" sounds strange. Stored in?

Reply:
I would opt for "The pipe `|` character is replaced by the text that corresponds to the value property." see my suggestion